bin/create-exercise

```diff
@@ -32,6 +32,5 @@ read -p "What's the difficulty for ${slug}? " difficulty
 bin/fetch-configlet
 bin/configlet create --practice-exercise "${slug}" --author "${author}" --difficulty "${difficulty}"
 
-cp exercises/practice/hello-world/.gitignore exercises/practice/${slug}/.gitignore
```

Comment:
There is a reason why this was here in the first place and so removing this is likely to impact something.

Is their documentation to update that addresses what this change is going to make happen, either in the view of the student, or the view of the maintainers?

How does this change impact what happens in this project?

Reply:
I put that there, to automate making new exercises like the existing exercises.

Bur we don't have any reason why the existing exercises had .gitignore